src/llm_vm/client.py

```diff
@@ -61,7 +61,7 @@ def __init__(self, big_model = default_big_model, small_model =default_small_mod
 
         #     return self.teacher.generate(prompt, max_len,**kwargs)
 
-        def CALL_SMALL(prompt, max_len=256, **kwargs):
+        def CALL_SMALL(prompt, max_len=40, **kwargs):
```

Comment:
why this change?

Reply:
sorry my bad

Reply:
Don't sweat it!  It's just an annoying necessity to keep pull requests minimal and local for bookkeeping 😃